examples/scara_arm/README.md

```diff
@@ -0,0 +1,36 @@
+# Scara Robot - Kinetics and Dynamics Equations
+
+![Robot](img\2dscara.png)
```

Comment:
forward slash should be a backslash, or better yet, don't put the image in a subdirectory.

Reply:
The image subdirectory was removed.